lib/src/Uncertainty/Distribution/TruncatedDistribution.cxx

```diff
@@ -525,6 +525,22 @@ NumericalSample TruncatedDistribution::getSupport(const Interval & interval) con
   return distribution_.getSupport(getRange().intersect(interval));
 }
 
+/* Get the PDF singularities inside of the range - 1D only */
+NumericalPoint TruncatedDistribution::getSingularities() const
+{
+  NumericalPoint singularities(0);
+  NumericalPoint nontruncatedSingularities(distribution_.getSingularities());
+  const UnsignedInteger size = nontruncatedSingularities.getSize();
+  if (size == 0) return singularities;
```

Comment:
this test is redundant with the one in the loop : the body is not executed and singularities is returned anyway.